src/pybind/mgr/prometheus/module.py

```diff
@@ -1562,6 +1563,20 @@ def get_collect_time_metrics(self) -> None:
                 cast(MetricCounter, sum_metric).add(duration, (method_name,))
                 cast(MetricCounter, count_metric).add(1, (method_name,))
 
+    def get_pg_repaired_objects(self) -> None:
+        dump = self.get('pg_dump')
+        for stats in dump['pool_stats']:
+            path = f'pg_objects_repaired{stats["poolid"]}'
+            self.metrics[path] = Metric(
+                'counter',
+                'pg_objects_repaired',
+                'Number of objects repaired in a pool Count',
+                ('poolid',)
```

Comment:
@pereman2 Why is this metric specifies `poolid` label instead of `pool_id` like (all?) other metrics?

The ones that are closely related such as `num_objects_recovered` or `objects` certainly use `pool_id`.  It would be nice to be consistent.